### DISCUSSION TITLE
Publish OSP-core in pypi

### DISCUSSION THREAD

**Discussion by yoavnash** [simphony/simphony-osp#686] [General]
Publish OSP-core in pypi to enable easy installation of the package.

Options for the project name:
1. `pip install simphony`
2. `pip install osp-core`
3. `pip install simphony-osp-core`

Option 3 has the advantage of being able to extend for wrappers. For example: `pip install simphony-wrapper-simlammp`.

Relevent links:
- For testing: https://test.pypi.org/
- For registering: https://pypi.org/

**Comment by pablo-de-andres** [comment]
We could also use the first option and extend it for wrappers, right? `simphony` installs the core, the rest come with the prefix.
Might be simpler.

**Comment by pablo-de-andres** [comment]
[simphony](https://pypi.org/project/simphony/) is taken.

**Comment by yoavnash** [comment]
Option 1:
```
simphony-osp-core
simphony-wrapper-lammps
simphony-wrapper-gromacs
```

Option 2:
```
simphony-osp
simphony-osp-lammps
simphony-osp-gromacs
```

**Selected: option 2**

**Comment by domgz** [comment]
As this is finally going to be done, we can move the discussion to an actual [issue](https://github.com/simphony/osp-core/issues/733). Therefore, I am locking this conversation.
